### PR TITLE
Add notes for calculus, statistics and probability

### PR DESCRIPTION
**Pull request by subinasr**

### CODE REVIEW

File: week1/calculus.ipynb [hunk 1]
```diff
@@ -0,0 +1,1060 @@
+{
```
Comment (ranjan-stha) [inline week1/calculus.ipynb:1]: <p>Also, mention the application of Differentiation in Neural Networks. For e.g. Differentiation is used to see the rate of change in the loss when a small change is made in weights/bias in the X layer in backward pass.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/4/discussion/#comment-1628771283'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ranjan-stha) [inline week1/calculus.ipynb:1]: <p>Also, mention that the chain rule is used in the Backpropagation step in Neural network to see the rate of change on some weight keeping all other weights/bias constants. This is the important step through which we can find rate of change in each neuron and helps to tune parameters for learning purposes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/4/discussion/#comment-1628771285'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>
